src/components/staking/DelegationsOverview.vue

```diff
@@ -49,7 +48,18 @@ export default {
     delegations: []
   }),
   computed: {
-    ...mapGetters(["address", `network`])
+    ...mapGetters(["address", `network`, `networks`])
+  },
+  methods: {
+    goToValidators() {
+      this.$router.push({
+        name: "Validators",
+        params: {
+          networkId: this.networks.find(network => network.id === this.network)
```

Comment:
thinking: network slug should be a getter (another PR)

Reply:
also we should rename that param. `networkId` is too confusing. It should be `networkSlug`